### DISCUSSION TITLE
Dark mode for React Query docs

### DISCUSSION THREAD

**Discussion by himankpathak** [TanStack/query#3511] [Ideas]
Can we implement a dark mode toggle for react query docs?
Something similar to how it's implemented in [Recoil](https://recoiljs.org/) or [Redux ](https://redux.js.org/)docs.

**Comment by TkDodo** [comment]
they are dark already. are you saying there are developers that want _light_ devtools 😮 ?

**Reply by himankpathak** [reply]
I am referring to [docs](https://react-query.tanstack.com/reference/useQuery) not devtools.
`https://github.com/tannerlinsley/react-query/tree/master/docs`

**Reply by TkDodo** [reply]
oh, I misread, sorry 🙈 

I tried to do that once, with not much success. Tbh, I would really like a docs rewrite to something like docosaurus, with more out of the box support. But we'd have to align that with the whole TanStack docs, so it's a larger issue. We can likely do it after we're in the monorepo - maybe there will even be "new docs" where the docs for all libraries of the TanStack live.

**Reply by himankpathak** [reply]
Docusaurus would definitely be nice! It supports mdx out of the box so shouldn't be much rewrites in migrating.👍 
Maybe we can incrementally migrate TanStack docs starting with react-query?

**Reply by TkDodo** [reply]
we'd have to see what @tannerlinsley has to say about this. One thing I would be excited to try is [remark-typescript-tools](https://github.com/phryneas/remark-typescript-tools) that would allow all code examples to be written in TS, and can be shown by the user in either JS or TS. They would also be compiled, too, to make sure that all changes in the code are actually updated in the example as well.

**Comment by DavNej** [comment]
Hi @TkDodo,
could you indicate to me how the docs is generated from https://github.com/TanStack/query/tree/main/docs ?
I don't understand how/where the `.md` files are parsed, and I can't find it.

**Reply by TkDodo** [reply]
https://github.com/TanStack/tanstack.com
